aioesphomeapi/client.py

```diff
@@ -1154,6 +1155,35 @@ def lock_command(
             req.code = code
         self._get_connection().send_message(req)
 
+    def valve_command(
+        self,
+        key: int,
+        position: float | None = None,
+        stop: bool = False,
+    ) -> None:
+        connection = self._get_connection()
+        req = ValveCommandRequest(key=key)
+        apiv = self.api_version
+        if TYPE_CHECKING:
+            assert apiv is not None
+        if apiv >= APIVersion(1, 1):
```

Comment:
We shouldn't need an api version check since this is new

Reply:
Done...